ARKitExample/ViewController.swift

```diff
@@ -103,11 +103,11 @@ class ViewController: UIViewController {
             return
         }
 
-        // As a TVIVideoSource, we must deliver CVPixelBuffers and not CGImages to the consumer.
+        // As a VideoSource, we must deliver CVPixelBuffers and not CGImages to the consumer.
```

Comment:
Not using first person "we":

// A VideoSource must deliver CVPixelBuffers (and not CGImages) to the VideoSink.

Reply:
Done